docs/index.rst

```diff
@@ -84,17 +84,17 @@ User Documentation
    io/ascii/index
    io/votable/index
    io/misc
+   vo/index
```

Comment:
Yes, I agree it should be moved. But it is more like a query using VO standards than I/O.